Automattic/jetpack
# Boost: remove WP Super Cache promos from settings page

Remove the WP Super Cache promo components from the Boost settings page. We want people to use Boost to cache sites now.

Fixes #36688

## Proposed changes:

* Removed the projects/plugins/boost/app/assets/src/js/features/super-cache-info/ directory and everything in it.
* Stop loading that code in projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply PR
* Make sure WP Super Cache is installed and configured, with or without caching enabled.
* Load up the Boost setting page (may need to shift-refresh to force reload)
* Confirm that the WPSC promo doesn't appear at the end of the page.
